advisories/0011.md

```diff
@@ -0,0 +1,43 @@
+### Mirantis Kubernetes Engine (MKE) and CVE-2024-21626: Understanding the Impact and Mitigation
+#### Release Date
+2024-02-08
+#### Overview
+This advisory addresses the impact of CVE-2024-21626, a vulnerability identified within `runc`, and its implications for the Mirantis Kubernetes Engine (MKE). Given that MKE relies heavily on the Mirantis Container Runtime (MCR), this vulnerability directly impacts its operational integrity. We aim to clarify the situation, explain the mitigation steps, and provide actionable guidance for MKE users.
+
+### Vulnerability Information
+#### CVE Identifier
+CVE-2024-21626
+#### CVSSv3.1
+8.6 (High) CVSS:3.1/AV:L/AC:L/PR:N/UI:R/S:C/C:H/I:H/A:H
+#### CWEs
+CWE-403, CWE-668
+
+#### Affected Product
+* MKE deployment requires installation of MCR.  Older versions of MCR (<= 23.0.9) contain the vulnerability in the underlying `runc` (<= 1.1.11-rc1) component. Please see the [MCR advisory](https://github.com/Mirantis/security/blob/main/advisories/0010.md) for details.  MKE deployments with MCR <= 23.0.9 require an upgrade to MCR 23.0.9-1 or later to patch `runc` to 1.1.11-rc1.m1.
+
+* No MKE Image Update Needed
+  
+   While CVE-2024-21626 might appear in vulnerability scans of your MKE images due to the dependent library, an update to MKE images is not currently required.  We understand this discrepancy can be confusing. To address this and enhance transparency, we are actively working to remove unnecessary references to the vulnerability from future MKE image scans. This update is planned for upcoming MKE releases.
+
+#### How to check if MKE is vulnerable
+1. [Download and config the client bundle](https://docs.mirantis.com/mke/3.7/ops/access-cluster/download-configure-client-bundle.html)
+2. Check the `runc` version by
+   ```
+   docker version --format "{{range .Server.Components}}{{if eq .Name \"runc\"}}{{.Version}}{{end}}{{end}}"
```

Comment:
Wouldn't that only check the runc version installed on one of the manager nodes? Or does MKE do some magic in the Engine API proxy? How would a user check the rest of the nodes in their cluster?

Reply:
`docker node ls --format {{.EngineVersion}}` to check engine version.  There is no viable way other than `ssh` into nodes.